pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -35,8 +35,8 @@ let
   targetPrefix = stdenv.lib.optionalString (targetPlatform != hostPlatform)
                                            (targetPlatform.config + "-");
 
-  ccVersion = (builtins.parseDrvName cc.name).version;
-  ccName = (builtins.parseDrvName cc.name).name;
+  ccVersion = cc.version or (builtins.parseDrvName cc.name).version;
```

Comment:
How about
```suggestion
  ccVersion = lib.getVersion cc;
```

Reply:
Good point. I wonder if we should add a `lib.getPName` as well?

Reply:
@gloaming I think that this goes out of scope of this PR.